test/envoye2e/README.md

```diff
@@ -0,0 +1,173 @@
+# Integration Test for WASM Filter
```

Comment:
This looks great! Though this looks more like a very detailed explanation of the attributegen filter, which will be useful when someone want to understand how to write a test specifically. Could you move it to stats_plugin folder?